test/integration/targets/web3-1.0.0/DumbContract.spec.web3.ts

```diff
@@ -23,16 +22,16 @@ describe("DumbContract", () => {
     const contract = await deployContract<DumbContract>("DumbContract");
 
     await contract.methods.countup(2).send({ from: accounts[0] });
-    expect(await contract.methods.counter().call()).to.be.eq("2");
+    expect((await contract.methods.counter().call()).toString()).to.be.eq("2");
     await contract.methods.countup("2").send({ from: accounts[0] });
-    expect(await contract.methods.counter().call()).to.be.eq("4");
+    expect((await contract.methods.counter().call()).toString()).to.be.eq("4");
   });
 
   it("should allow to pass signed values in multiple ways", async () => {
     const contract = await deployContract<DumbContract>("DumbContract");
 
-    expect(await contract.methods.returnSigned(2).call()).to.be.eq("2");
-    expect(await contract.methods.returnSigned("2").call()).to.be.eq("2");
+    expect((await contract.methods.returnSigned(2).call()).toString()).to.be.eq("2");
```

Comment:
according to new typings this should be BN, right? Could you add some kind of type assertion as well?

Reply:
Good idea, I've now done this in [44e6062](https://github.com/ethereum-ts/TypeChain/commit/44e6062a500abcc46af0562736f42367bf243f53) .

It looks like there's still some BigNumber/BN confusion to sort out in web3: https://github.com/ethereum/web3.js/issues/2468